README.md

```diff
@@ -60,10 +60,11 @@ Maven Execution Research [maven execution research](https://github.com/ClintonCa
 
 Testing Documentation [testing documentation](https://github.com/ClintonCao/Contextproject-TSE/blob/master/documentation/Testing%20related%20documents/TestingDocumentation.md)
 
-## Link to SIG Feedback 1 (Picture):
-SIG Feedback: [SIG Feedback](https://github.com/ClintonCao/Contextproject-TSE/blob/master/documentation/SIG%20Feedback/SIGFeedback.JPG)
+## Link to SIG Feedback & Code Improvements:
+The feedback of SIG and the code improvements can be found [here](https://github.com/ClintonCao/Contextproject-TSE/tree/master/documentation/SIG%20Feedback).
```

Comment:
It does not take you to the picture anymore, but the whole folder containing the feedback.